package.json

```diff
@@ -62,11 +63,14 @@
     "rollup-plugin-string": "^3.0.0",
     "rollup-plugin-swc": "^0.2.1",
     "serve": "^14.1.1",
+    "sharp": "^0.32.6",
     "typedoc": "^0.23.24",
     "typedoc-plugin-markdown": "^3.14.0",
-    "typescript": "^4.8.3"
+    "typescript": "^4.8.3",
+    "vitest": "^0.34.6"
   },
   "dependencies": {
-    "@types/geojson": "^7946.0.10"
+    "@types/geojson": "^7946.0.10",
+    "web-merc-projection": "^1.3.2"
```

Comment:
I feel like importing this whole library just for a Point definition and a wgs84 to mercator conversion may not be necessary. Wouldn't it be possible instead to have those bits in the geometry/utils.ts ?